### PR TITLE
refactor: move more os discovery to yaml files

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Moved more os discovery into yaml config files.

### CODE REVIEW

File: includes/definitions/mgeups.yaml [hunk 1]
```diff
@@ -7,3 +7,10 @@ mib_dir:
     - mge
 over:
     - { graph: device_current, text: Current }
+discovery:
+    - sysDescr_regex:
+        - '/Pulsar M/'
+        - '/MGE UPS SYSTEMS - Network Management Proxy/'
```
Comment (murrant) [inline includes/definitions/mgeups.yaml]: Again move the regular ones to sysDescr:

**Comment by laf** [comment]
@murrant phpunit fails when those changes are made - doesn't it cause it to need both of those to match:

```bash
Running unit tests... failed
PHPUnit 4.8.31 by Sebastian Bergmann and contributors.

...............................................................  63 / 266 ( 23%)
.........F..................................................... 126 / 266 ( 47%)
........................F...................................... 189 / 266 ( 71%)
............................................................... 252 / 266 ( 94%)
..............

Time: 31.15 seconds, Memory: 6.00MB

There were 2 failures:

1) LibreNMS\Tests\DiscoveryTest::testDlink
Test file: dlink-des.snmprec

Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'dlink'
+'generic'

/opt/librenms/tests/OSDiscoveryTest.php:46
/opt/librenms/tests/OSDiscoveryTest.php:453

2) LibreNMS\Tests\DiscoveryTest::testMgeups
Test file: mgeups-pulsar.snmprec

Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'mgeups'
+'generic'

/opt/librenms/tests/OSDiscoveryTest.php:46
/opt/librenms/tests/OSDiscoveryTest.php:897

FAILURES!
Tests: 266, Assertions: 1275, Failures: 2.
snmpsimd: output at /tmp/snmpsimd.logRunning style check... success
Running lint check... ^C
-bash-4.2$ cat includes/definitions/dlink.yaml
os: dlink
text: 'D-Link Switch'
type: network
icon: dlink
ifname: 1
discovery:
    - sysDescr:
        - '/D-Link DES-/'
        - '/Dlink DES-/'
    - sysDescr_regex:
        - '/^DES-/'
        - '/^DGS-/'
-bash-4.2$ cat includes/definitions/mgeups.yaml
os: mgeups
text: 'MGE UPS'
group: ups
type: power
icon: mge
mib_dir:
    - mge
over:
    - { graph: device_current, text: Current }
discovery:
    - sysDescr:
        - '/Pulsar M/'
        - '/MGE UPS SYSTEMS - Network Management Proxy/'
    - sysDescr_regex:
        - '/^Galaxy/'
        - '/^Evolution/'
        - '/^Comet/'
-bash-4.2$
```

**Comment by murrant** [comment]
No, you forgot to remove the `/`'s :)

**Comment by laf** [comment]
:(

**Comment by laf** [comment]
Updated

**Pull request merged**